### PR TITLE
I changed various bugs but mostly the assessments

### PR DESCRIPTION
**Pull request by xd-arsenic**
Assessments
Assessment Review
Comment Replies
Message background
Google Docs Embedded Page
Fixed No Content Background For Mastery
Other small fixes 

image

image

image

image

image

image

ENTER ISSUES HERE

### CODE REVIEW

**Comment by Roguim** [comment]
nice 👌

File: js/all.js [hunk 1]
```diff
@@ -1009,4 +1009,8 @@ async function createQuickAccess() {
     }
 }
 
-Logger.log("Finished loading all.js");
\ No newline at end of file
+Logger.log("Finished loading all.js");
+
+
+
+
```
Comment (glen3b) [inline js/all.js:1016]: Why the added multiple newlines?

File: css/modern.css [hunk 2]
```diff
@@ -1755,9 +1765,664 @@ table#collection-view-contents tbody tr td {
...
+            .lrn_response.lrn_clearfix.lrn_style_button > .lrn-response-validate-wrapper.lrn_response_input.lrn_incorrect {
+                background-color: var(--accent) !important;
+            }
+
+            .lrn-response-validation-wrapper > .lrn_correctAnswers  {
+                background-color: var(--active) !important;
+            }
+
+
+
```
Comment (glen3b) [inline css/modern.css:2311]: Can you de-indent these? If this is deliberate can you comment why?

File: css/modern.css [hunk 3]
```diff
@@ -1755,9 +1765,664 @@ table#collection-view-contents tbody tr td {
     background-color: var(--highlight) !important;
 }
 
+
+/* Quizes */
```
Comment (glen3b) [inline css/modern.css:1769]: Spelling. Also, "modern" UI quizzes or old quizzes (please clarify)?

**Review by glen3b — COMMENTED** [review]
I haven't done a thorough review, but based on a quick pass, here are a few quick things that can be tweaked.

File: css/modern.css (hunk 2 above)
Comment (xd-arsenic) [reply css/modern.css:2311]: It's part of the entire Assessment Section

File: js/all.js (hunk 1 above)
Comment (xd-arsenic) [reply js/all.js:1016]: hmm interesting question. i dont know. lol

File: css/modern.css (hunk 3 above)
Comment (xd-arsenic) [reply css/modern.css:1769]: I believe it is old quizes

**Review by aopell — APPROVED** [review]
I made a bunch of changes

**Pull request merged**